test/kubeclient_test.rb

```diff
@@ -1,7 +1,4 @@
-require 'minitest/autorun'
-require 'json'
-require 'webmock/minitest'
-require './lib/kubeclient'
+require './test/test_helper'
```

Comment:
> Rake::TestTask.new do |t| t.libs << "test" end

@Fryguy thanks for all the help.
I tried to do the above in Rakefile, and switched to use require 'test_helper' in files, however now when running rake test there are 0 tests running.
(0 runs, 0 assertions, 0 failures, 0 errors, 0 skips)
I'm checking Rake docs to see what's missing...